src/main/java/com/uipath/uipathpackage/UiPathPack.java

```diff
@@ -261,5 +271,9 @@ public FormValidation doCheckOutputPath(@QueryParameter String value) {
             }
             return FormValidation.ok();
         }
+
```

Comment:
remove all these unnecessary lines from here and any other place

Reply:
Removed redundant lines, keeping line spaces between methods for readability purposes